general_considerations.md

```diff
@@ -65,6 +77,9 @@ Transaction` transaction is included in the `Merkle Root`. We assume that this
 [consensus mechansim](#consensus-mechanism), for instance in an `OP_RETURN`
 output.
 
+[kp] We are getting into too many details here. The reader has no clue
+about why we are using P2TR.  [/kp]
+
```

Comment:
P2R is required for FROST/ROAST and this doc should specify at that level of detail, though I think at this point in the doc is not the correct place to explain FROST. (which should come below)